usaspending_api/etl/rapidloader.py

```diff
@@ -28,6 +29,15 @@ def run_load_steps(self) -> None:
         download_queue = Queue()  # Queue for jobs which need a csv downloaded
         es_ingest_queue = Queue(20)  # Queue for jobs which have a csv and are ready for ES ingest
 
+        updated_record_count = get_updated_record_count(self.config)
+        printf(
+            {
+                "msg": "Found {} new {} records to update in ElasticSearch".format(
```

Comment:
f strings are preferable now since we're using Python3.7, it might allow this to be a single line.

Minor gripe: when creating a new index the log state says "update"

Reply:
Additionally, might as well format the number to include comma separators since we expect every load to be over 1K and often orders of magnitude larger